src/liballoc/sync.rs

```diff
@@ -311,6 +311,37 @@ impl<T> Arc<T> {
         Self::from_inner(Box::into_raw_non_null(x))
     }
 
+    /// Construct a Arc box with uninitialized contents.
```

Comment:
```suggestion
    /// Constructs a new `Arc` with uninitialized contents.
```
For consistency with Rc.